de/installation/README.md

```diff
@@ -1,16 +1,22 @@
 # Wenn du dieses Tutorial zu Hause bearbeitest
 
-Wenn du das Tutorial zu Hause bearbeitest, und nicht auf einer der ["Django Girls"-Veranstaltungen](https://djangogirls.org/events/), kannst du dieses Kapitel komplett überspringen und direkt mit dem Kapitel [Wie funktioniert das Internet?](../how_the_internet_works/README.md) fortfahren.
```

Comment:
What was the reason to remove the quotation marks and the hyphen here? In German you can't just chain words with spaces, you have to hyphenate them or merge them into a single word.

Reply:
No deep reason. It was written without quotation marks at some other place in the tutorial, so I just wanted to harmonize that, to have one style. And Django Girls has never been in quotation marks before also, so I decided to take that style without the quotation marks. Sorry, we can of course change that again.

Reply:
No need to apologize. Harmonizing orthography of different usages of the same term is a good idea.

To not highjack this already-merged PR to much, I've filed a new issue where we can discuss on how to harmonize these phrases: #1421